index.Rmd

```diff
@@ -5,14 +5,14 @@ subtitle: Visualization and Classification for Larceny, Assault and Harassment
 output: html_document
 ---
 
-# Introduction
+# Introduction 
```

Comment:
The introduction is very clear about what you are doing. I would be more scientific introduction if you could add an objective or a hypothesis of your study.